examples/disaggregated/simpler_example/run.sh

```
@@ -0,0 +1,63 @@
+    --config ./gen_config.yaml 2>&1  | tee log_gen_0 &
+
+sleep 80
+echo "Starting disaggregated server"
+# Start disaggregated server
+trtllm-serve disaggregated -c ./disagg_config.yaml 2>&1 | tee log_disagg &
+
+echo "Waiting for the disaggregated server to start"
+# wait for the disaggregated server to start
+
+# Health check for the disaggregated server
+while [ $(curl -s -o /dev/null -w "%{http_code}" "http://localhost:8000/health") -ne 200 ]; do
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)